src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -93,6 +94,20 @@ public void savesAndLoads() throws Exception {
         );
     }
 
+    @Test
+    public void saveOverwrites() throws Exception {
+        final byte[] original = "1".getBytes();
```

Comment:
@olegmoz Lets use a constant for this values

Reply:
@paulodamaso there is no reason to bring this value outside of test, it is not going to be used anywhere else

Reply:
@olegmoz Yes, you are right